lib/helpers.js

```diff
@@ -165,6 +172,64 @@ export async function openFirstPartyWindow(event) {
 
 export const prettify = obj => JSON.stringify(obj, null, 2);
 
+export async function toNFCPayload({credential}) {
+  const nfcRenderingTemplate2024 = _getNFCRenderingTemplate2024({credential});
+  const bytes = await _decodeMultibase(nfcRenderingTemplate2024.payload);
+  return {bytes};
+}
+
+export function hasNFCPayload({credential}) {
+  try {
+    const nfcRenderingTemplate2024 = _getNFCRenderingTemplate2024({credential});
+    if(!nfcRenderingTemplate2024) {
+      return false;
+    }
+
+    return true;
+  } catch(e) {
+    return false;
+  }
+}
+
+function _getNFCRenderingTemplate2024({credential}) {
+  let {renderMethod} = credential;
+  if(!renderMethod) {
+    throw new Error('Credential does not contain "renderMethod".');
+  }
+
+  renderMethod = Array.isArray(renderMethod) ? renderMethod : [renderMethod];
+
+  let nfcRenderingTemplate2024 = null;
+  for(const rm of renderMethod) {
+    if(rm.type === 'NfcRenderingTemplate2024') {
+      nfcRenderingTemplate2024 = rm;
+      break;
+    }
+    continue;
+  }
+
+  if(nfcRenderingTemplate2024 === null) {
+    throw new Error('Credential does not support "NfcRenderingTemplate2024".');
+  }
+
+  if(!nfcRenderingTemplate2024.payload) {
+    throw new Error('NfcRenderingTemplate2024 does not contain "payload".');
+  }
+
+  return nfcRenderingTemplate2024;
+}
+
+async function _decodeMultibase(input) {
+  const multibaseHeader = input.slice(0, 1);
```

Comment:
No need to copy the string, just grab the char. Dunno if this optimizes to the same thing of course.

```suggestion
  const multibaseHeader = input[0];
```